frontend.java

```diff
@@ -0,0 +1,540 @@
+						*/
+						if(logInCom.equals("login")){//this doesnt work. 
+							loggedIn = 1;
+							System.out.println("Logged In");	
+						}else{
+							System.out.println("Invalid log in.");
+						}
+					}
+						
+					
+					/*
+				    String logInCom = input();
+				    System.out.println(logInCom);//Test
+				    
+				    if(logInCom != "login"){
+				    	System.out.println("Invalid command, please log in");
+				    	System.exit(1);//Shouldnt exit, simply reset the command process
+				    
+				    }else{
+				    */
+				    if(loggedIn == 1){
+				    	//Looking for type of user.
+				    	System.out.println("Please enter a type of user. (Retail or Agent): Command>");
+				    	String logInTypeCom = input();
+				    	System.out.println(logInTypeCom);
+				    	
+				    	if(logInTypeCom.equals("retail")){
+				    		loggedIn = 1;
+				    		type= 1;
+				    		takeCommand();
+				    	}else if(logInTypeCom.equals("agent")){
+				    		loggedIn = 1;
+				    		type= 2;
+				    		takeCommand();
+				    	}else{
+				    		System.out.println("Invalid type, please log in");
+					    	System.exit(1);//Shouldnt exit, simply reset the command process
+				    	}
+				    }
+				}else{
+					//Logged In
+					takeCommand();
+				}
+	}
+	public static String input(){
+		BufferedReader br = new BufferedReader(new InputStreamReader(System.in)); 
+	    String command = null;
+	 
+	    //  read the username from the command-line; need to use try/catch with the
+	    //  readLine() method
+	    try {
+	    	command = br.readLine();
+	    } catch (IOException ioe) {
+	    	//Set up valid error message
+	    	System.out.println("IO error trying to read your name!");
+	        System.exit(1);//Shouldnt exit, simply reset the command process
+	    }
+	    command = command.toLowerCase();
+		return command;
+	}
+	public static void takeCommand(){
+		System.out.print("Command>");
+		String command = input();
+		if(command.equals("login")){
+			error("You are already logged in.");
+		}else if(command.equals("logout")){
+			front_logout();
+		}else if(command.equals("create")){
+			if(type == 2){
+				front_create();
+			}else{
+				error("Not logged into the proper account.");
+			}
+		}else if(command.equals("delete")){
+			if(type == 2){
+				front_delete();
+			}else{
+				error("Not logged into the proper account.");
+			}
+		}else if(command.equals("deposit")){
+			front_deposit();
+		}else if(command.equals("withdraw")){
+			front_withdraw();
+		}else if(command.equals("transfer")){
+			front_transfer();
+		}else{
+			error("Some error statement.");
+		}
+	}
+	public static void error(String errorMessage){
+		System.out.println(errorMessage);
+		takeCommand();
+	}
+	public static void front_create() {
+		// get the account number
+		// get the account name
+		// update the transaction summary file
+		System.out.print("Account Number:");
+		String accountNumber = frontend.input();
+		if(accountNumber(accountNumber)==false){
```

Comment:
This is a very confusing combination of function name and variable name. There's no way to know what this function may do until you look at the implementation.